src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -1612,6 +1613,16 @@ public SyntaxNode DottedName(string dottedName)
         public abstract SyntaxNode TypeExpression(ITypeSymbol typeSymbol);
 
         /// <summary>
+        /// Creates an expression that denotes a type.
+        /// Adds a <see cref="DoNotAddImportsAnnotation"/> which will prevent any
```

Comment:
This feels weird.  Why do we need a DoNotAddImports annotation?  Why not just not put on the annotation that will cause an import to be added?